pkg/shares/utils.go

```diff
@@ -93,3 +93,13 @@ func zeroPadIfNecessary(share []byte, width int) (padded []byte, bytesOfPadding
 	share = append(share, padding...)
 	return share, missingBytes
 }
+
+// contains returns true if slice contains element.
+func contains(slice []byte, element byte) bool {
```

Comment:
[Non Blocking][Suggestion]
Why not use https://pkg.go.dev/bytes#Contains ?

Reply:
Good idea. Done in https://github.com/celestiaorg/celestia-app/pull/837/commits/9a0d60e2d6be8fd82a45086f3b3a08a480be971c